modules/codelite/codelite.lua

```diff
@@ -26,10 +26,19 @@
 		return cfgname
 	end
 
+	-- Element text is not escaped the same as element attributes
+	function codelite.escElementText(value)
+		local result = value:gsub('&', '&amp;')
+		result = result:gsub('<', '&lt;')
+		result = result:gsub('>', '&gt;')
+		return result
+	end
+
 	function codelite.esc(value)
 		local result = value:gsub('&', '&amp;')
 		result = result:gsub('<', '&lt;')
 		result = result:gsub('>', '&gt;')
+		result = result:gsub('"', '&quot;')
```

Comment:
I wanted to err on the side of caution with this, the escaping seems to be very fragile and inconsistent in CodeLite and I wanted to avoid future issues. For example, not escaping quotes results in the cURL project to silently fail to load in the Premake workspace - I believe there's another project that fails too.